draft-ralston-mimi-terminology.md

```diff
@@ -165,52 +163,52 @@ than a function call if the client and server are the same logical entity.
+is accepted or whether the user is able to become a room participant.
 
-**Admin Permissions**: Typically at least the user who created the chat, these permissions
-grant the associated users an ability to change the permissions of other users in the chat.
-The set of users with these permissions in a chat are called **Admins**. Admin permissions
-inherit moderator permissions.
+**Admin Permissions**: Typically at least the user who created the room, these permissions
+grant the associated users an ability to change the permissions of other users in the room.
+The set of users with these permissions in a room are called **Admins**. Admin permissions
+typically inherit moderator permissions.
 
 **Moderator Permissions**: These permissions grant the associated users an ability to kick,
-ban, or otherwise restrict another user's ability to use the chat. For example, deleting
-a spammer's events. The set of users with these permissions in a chat are called **Moderators**.
+ban, or otherwise restrict another user's ability to use the room. For example, deleting
+a spammer's events. The set of users with these permissions in a room are called **Moderators**.
 
 Note that with both moderator permissions and admin permissions a system may have finer
 granularity, such as a set of users being able to kick but not ban. Documents with these
 semantics should clarify this case.
 
-**Invite**: An action taken by a user in a chat to encourage another user to become a
-chat member (or joined to the chat). This can be explicit through the server-server API,
-or implicit with an invite link/join code.
+**Invite**: An action taken by a user in a room to encourage another user to become a
+room participant (or joined to the room). This can be explicit through the server-server API,
+or implicit with an join link/join code.
 
-**Invite Link**: An out-of-band invite for a user to join the chat, represented as a URL
+**Join Link**: A machine-readable mechanism for a user to join the room, represented as a URL
 or QR code.
 
-**Join Code** or **Join Password**: A text-based string a user may enter to join a chat.
+**Join Code** or **Join Password**: A text-based string a user may enter to join a room.
 The string may be shared with the user verbatim or encoded with a QR code, for example.
 
 **Direct Invite**: An invite to a DM or Group DM. These invites might appear in a different
 section of the client's UI to denote their semantic difference from a non-DM invite.
 
-**Kick**: An action taken by a user in a chat to remove another user, assuming the sender
-has moderator permissions. The affected user can re-join the chat with either another invite
-or by simply joining, depending on the chat type.
+**Kick**: An action taken by a user to remove another user from a room, assuming the sender
+has moderator permissions. The affected user can re-join the room with either another invite
+or by simply joining, depending on the room type.
```

Comment:
I don't think we have consensus on out-of-band policy interactions, so would defer to leaving this as-is for now. Possibly adding a TODO to denote it's fluid nature.

Reply:
Your wording explicitly rules it out of scope, whereas my proposed wording allow us to defer such a decision.